### PR TITLE
Man

### PR DESCRIPTION
**Pull request by guynan**
This is a more comprehensive (*though not complete*) iteration of the `kodev` manpage. I would like to add some more examples however I cannot quite think what.

There are references to `KORE(3)` which will be resolved once you are comfortable with the format.

### CODE REVIEW

File: share/man/kodev.1 [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+where these functions and structures are described.
+.BR kore
+projects may be managed using the following OPTIONS;
+
+.BR create
+.RS
+Create a new application skeleton with the name that is passed to it. This will
+create a new directory with all the files required to begin hacking about.
+This will also generate a self-signed certificate \fIunless\fR \fBkore\fR was
+built with \fBNOTLS=1\fR. Note this certificate should not be used for
```
Comment (jorisvink) [inline share/man/kodev.1]: This isn't true.

Kodev will always generate the self-signed certs / write out the DH parameters unless single-binary is turned on in the build.conf and NOTLS=1 is set in the kore_flavor option.

File: share/man/kodev.1 [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+.BR build
+.RS
+Build the application. The properties of this build are read from
+.BR conf/build.conf
+and the semantics of which are discussed in \fBCONFIGURATION FILES\fR
+.RE
+
+.BR run
+.RS
+Run an application (\-fnr implied). The current directory is the path of the
```
Comment (jorisvink) [inline share/man/kodev.1]: I think we should clarify that kodev run will either run the kore binary (from PREFIX) or the application binary (if single-binary was turned on).

File: share/man/kodev.1 [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+project to be run. This will read from the \fBconf/app.conf\fR discussed in
+\fBCONFIGURATION FILES\fR unless otherwise specified. This will build the
+application\-\-if necessary\-\-and start it in the foregound. \fB^C\fR
+will stop the process.
+.RE
+
+.BR reload
+.RS
+Reload the application (SIGHUP sent). Reads the application run configuration
+which would be read in a normal run on a reload.
```
Comment (jorisvink) [inline share/man/kodev.1]: So SIGHUP actually does not reload the configuration, it tells kore to reload all loaded modules.

**Comment by jorisvink** [comment]
Starting to look very good, I left some comments on certain sections.

One important thing: the manual page doesn't properly render in 80 columns. It would be great if this could be fixed :)

**Comment by guynan** [comment]
Hi there, I will fix those comments. Though what do you mean by it doesn't render properly? I always use 80 char width terminals out of habit and it looks fine, unless I'm missing something?

**Comment by jorisvink** [comment]
On Linux it seems fine yes.

On the BSDs (including macos) not so much:

man

I'm sure it's an straight forward fix.

**Comment by guynan** [comment]
Ahh fantastic. I'll be able to isolate that quickly, probably.

**Comment by guynan** [comment]
Actually just to gather a few more details, which BSD are you trying this on? I'll spin up a VM later. 

Also I'm thinking I'll have some time to start the Kore man page and thinking I'll model it on the Linux programmers manual entry for string.

**Comment by jorisvink** [comment]
Actually I seem to have overextended a bit, it looks crap on macos only.

**Comment by guynan** [comment]
Hmm I will have to see what *roff they use.

**Comment by guynan** [comment]
Actually, could this be a tmux/screen issue?

**Comment by jorisvink** [comment]
Yup. It was, good catch. Ignore my ranting :)

**Pull request merged**